app/experimenter/experiments/constants/nimbus.py

```diff
@@ -175,6 +175,17 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_MOBILE_FIRST_RUN = NimbusTargetingConfig(
+    name="First start-up users",
+    slug="first_run_mobile",
+    description=(
+        "First start-up users (i.e first run after installing the app) on mobile"
+    ),
+    targeting=("isFirstRun == 'true'"),
+    desktop_telemetry="",
+    application_choice_names=(Application.FENIX.name, Application.IOS.name),
```

Comment:
Not Klar or Focus?

Reply:
Nope not yet, they don't consume the SDK (as far as I can tell, but either way the work we are doing is targeting iOS and Fenix first 😄)